docs/guidelines/releases-guidelines/03-kyma-cli-release-process.md

```diff
@@ -53,3 +53,13 @@ A Kyma CLI release consists of:
       git push --delete upstream {RELEASE_VERSION}
       git push upstream {RELEASE_VERSION}
       ```
+7. After the release tarball is available on Github, update the Homebrew formula, either automatically or manually:
+    - If no changes to the build command are needed, this can be done automatically by the Homebrew CLI:
+
+       ```bash
+       brew bump-formula-pr --strict kyma-cli --url https://github.com/kyma-project/cli/archive/{RELEASE_VERSION}.tar.gz
+       ```
+
+    - Alternatively, create a PR to the [kyma-cli Homebrew formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/kyma-cli.rb).
+
+    When a Homebrew maintainer approves your PR, the formula is updated.
```

Comment:
If this result is true for the manual and automatic option, keep it as is.
If it's only true for the 2nd option, we need more blank spaces (2 or 3?) so it visibly belongs to the 2nd bullet.

Reply:
It's true for both